### ISSUE TITLE
Problem converting files with MeshToVox

### ISSUE THREAD

**Issue by ehsanps4** [Zarbuz/FileToVox#67]
Hello. This problem has not been resolved yet.

https://github.com/Zarbuz/FileToVox/issues/58

scale: 1790

[file.zip](https://github.com/Zarbuz/FileToVox/files/9146332/file.zip)

![153490276-66e2ddb8-ead8-4392-9747-2d3250f5635b](https://user-images.githubusercontent.com/85092258/179892656-9397e465-547c-473e-8a87-6e4ce6a9b790.jpg)

![153490311-7ed0374f-5740-4ebd-9f04-8e2568ec2a0c](https://user-images.githubusercontent.com/85092258/179892688-d52b6fde-1d24-49c5-b2f8-ddec3755fcc9.jpg)

**Comment by Zarbuz** [comment]
Hello,

I finally managed to reproduce and fix your bug. I added a new option for version 2.8 to choose if the voxelization should be used with the GPU or the CPU. By default it will be the GPU but in your case where the number of vertices is just huge you will have to choose the CPU. Voxelization is a bit longer (depending on your hardware) but it will avoid missing parts.

![image](https://user-images.githubusercontent.com/4965714/185600987-1a8edcda-fee2-4ded-867c-e0dbc60edf15.png)
